code/datums/trading/traders/goods.dm

```diff
@@ -224,43 +224,44 @@ Sells devices, odds and ends, and medical stuff
-		/obj/item/destTagger                     = TRADER_THIS_TYPE,
-		/obj/item/gps                            = TRADER_THIS_TYPE,
-		/obj/item/measuring_tape                 = TRADER_THIS_TYPE,
-		/obj/item/ano_scanner                    = TRADER_THIS_TYPE,
-		/obj/item/core_sampler                   = TRADER_THIS_TYPE,
-		/obj/item/depth_scanner                  = TRADER_THIS_TYPE,
-		/obj/item/pinpointer/radio               = TRADER_THIS_TYPE,
-		/obj/item/stack/medical/advanced         = TRADER_BLACKLIST
+		/obj/item/flashlight                      = TRADER_ALL,
+		/obj/item/kit/paint                       = TRADER_SUBTYPES_ONLY,
+		/obj/item/aicard                          = TRADER_THIS_TYPE,
+		/obj/item/binoculars                      = TRADER_THIS_TYPE,
+		/obj/item/cable_painter                   = TRADER_THIS_TYPE,
+		/obj/item/flash                           = TRADER_THIS_TYPE,
+		/obj/item/paint_sprayer                   = TRADER_THIS_TYPE,
+		/obj/item/multitool                       = TRADER_THIS_TYPE,
+		/obj/item/lightreplacer                   = TRADER_THIS_TYPE,
+		/obj/item/megaphone                       = TRADER_THIS_TYPE,
+		/obj/item/paicard                         = TRADER_THIS_TYPE,
+		/obj/item/scanner/health                  = TRADER_THIS_TYPE,
+		/obj/item/scanner/breath                  = TRADER_THIS_TYPE,
+		/obj/item/scanner/gas                     = TRADER_ALL,
+		/obj/item/scanner/spectrometer            = TRADER_ALL,
+		/obj/item/scanner/reagent                 = TRADER_ALL,
+		/obj/item/scanner/xenobio                 = TRADER_THIS_TYPE,
+		/obj/item/suit_cooling_unit               = TRADER_THIS_TYPE,
+		/obj/item/t_scanner                       = TRADER_THIS_TYPE,
+		/obj/item/taperecorder                    = TRADER_THIS_TYPE,
+		/obj/item/batterer                        = TRADER_THIS_TYPE,
+		/obj/item/synthesized_instrument/violin   = TRADER_THIS_TYPE,
+		/obj/item/hailer                          = TRADER_THIS_TYPE,
+		/obj/item/uv_light                        = TRADER_THIS_TYPE,
+		/obj/item/organ/internal/brain_interface  = TRADER_SUBTYPES_ONLY,
+		/obj/item/robotanalyzer                   = TRADER_THIS_TYPE,
+		/obj/item/chems/toner_cartridge           = TRADER_THIS_TYPE,
+		/obj/item/camera_film                     = TRADER_THIS_TYPE,
+		/obj/item/camera                          = TRADER_THIS_TYPE,
+		/obj/item/destTagger                      = TRADER_THIS_TYPE,
+		/obj/item/gps                             = TRADER_THIS_TYPE,
+		/obj/item/measuring_tape                  = TRADER_THIS_TYPE,
+		/obj/item/ano_scanner                     = TRADER_THIS_TYPE,
+		/obj/item/core_sampler                    = TRADER_THIS_TYPE,
+		/obj/item/depth_scanner                   = TRADER_THIS_TYPE,
+		/obj/item/pinpointer/radio                = TRADER_THIS_TYPE,
+		/obj/item/stack/medical/ointment/advanced = TRADER_BLACKLIST,
+		/obj/item/stack/medical/bandage/advanced  = TRADER_BLACKLIST
```

Comment:
looking at the old code i think it just specifically blacklisted selling `/obj/item/stack/medical/advanced` because that's an abstract type that shouldn't be accessible? while the same isn't true for these two. `TRADER_BLACKLIST` is just specifically that type, while `TRADER_BLACKLIST_SUB` is that and subtypes.

it also seems like Kearel accidentally removed `/obj/item/stack/medical  = TRADER_SUBTYPES_ONLY` eight whole years ago, and without it blacklisting any subtypes is pointless anyway.